chapters/functions.tex

```diff
@@ -2057,7 +2058,7 @@ \subsubsection{Records}\label{records}
 \item
   The components of the Modelica record class are declared in the same order in the C struct.
 \item
-  Arrays cannot be mapped.
+  Records containing arrays cannot be mapped.
 \end{itemize}
 
 Records are passed by reference (i.e., a pointer to the record is being passed).
```

Comment:
This is where I think clarification for arrays is needed.  Something like the following might be enough:
> Scalar records are passed by reference (i.e., a pointer to the record is being passed).